docs/visualize/x-axis-aggs.asciidoc

```diff
@@ -6,10 +6,11 @@ Elasticsearch documentation for that aggregation.
 
 *Date Histogram*:: A {ref}/search-aggregations-bucket-datehistogram-aggregation.html[_date histogram_] is built from a
 numeric field and organized by date. You can specify a time frame for the intervals in seconds, minutes, hours, days,
-weeks, months, or years. You can also specify a custom interval frame by selecting *Custom* as the interval and
+weeks, months, or years. Intervals You can also specify a custom interval frame by selecting *Custom* as the interval and
```

Comment:
I can't seem to build the docs locally for some reason but is `Intervals` here supposed to be a new line or bolded syntax?